README.md

```diff
@@ -61,10 +62,6 @@ npm run dev
 
 5. Probot will then automatically create you a `.env` file in your local working directory (you can add `LOG_LEVEL=trace` for more verbose logging)
 
-6. From your browser, copy the installiation ID from the URL (e.g. at `https://github.com/settings/installations/123456` the ID is `123456`)
-
-7. Add `TAKO_INSTALLATION_ID` to your `.env` file, setting it's value to the ID you've just copied
-
```

Comment:
:tada:

Reply:
After this is merged, I'd like to spend an hour or so on documentation.